tasks/build/docker/docs/Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM alpine:3.6
```

Comment:
Maybe try `python:2.7-alpine` to see if it speeds it up any

Reply:
i didn't see anything major.  final image size was slightly bigger at 2.51gb vs 2.47.   i'm indifferent, want it added?